extensions/amp-story/1.0/amp-story.js

```diff
@@ -297,6 +297,12 @@ export class AmpStory extends AMP.BaseElement {
 
     /** @private {boolean} whether the styles were rewritten */
     this.didRewriteStyles_ = false;
+
+    /** @private @const {} */
+    this.subscriptionsService_ = null;
+
+    /** @private @const {} */
+    this.granted_ = true;
```

Comment:
Please be specific about what is granted, this is a 2500 LOC file

Reply:
Done.